cloudformation/main.yaml

```diff
@@ -28,7 +84,30 @@ Resources:
           Version: '2012-10-17'
         ManagedPolicyArns:
           - arn:aws:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole
-  
+        Policies:
+          - PolicyName: EmailTablePolicy
+            PolicyDocument:
+              Statement:
+                - Sid: 'EmailTable'
+                  Effect: 'Allow'
+                  Action: ['dynamodb:PutItem']
+                  Resource: 'arn:aws:dynamodb:*:*:table/Emails'
+              Version: '2012-10-17'
+
+
+  # Rule for cron job
+  ScheduledMailmanRule:
+    Type: AWS::Events::Rule
+    Properties:
+      ScheduleExpression: 'rate(1 minute)'
```

Comment:
We should bump this up to like 5/10 minutes so we don't overuse our free capacity :)

Reply:
Will fix!